Card/Tokenizer/Rsa.ts

```diff
@@ -10,7 +10,7 @@ export class Rsa extends Base {
 		return "0"
 	}
 
-	async getKeyName(): Promise<string> {
+	async getKeyName(encrypted?: cryptly.Encrypter.Rsa.Encrypted): Promise<string> {
```

Comment:
This parameter is never used?

Reply:
you're right, i can't remember why i added these arguments here but it looks like the type system is ok with removing them